politeiawww/backend.go

```diff
@@ -230,13 +230,6 @@ func (b *backend) validatePassword(password string) error {
 }
 
 func (b *backend) validateProposal(np www.NewProposal) error {
-	// Check for a non-empty name.
-	if np.Name == "" {
-		return userError{
-			errorCode: www.StatusProposalMissingName,
```

Comment:
`StatusProposalMissingName` isn't being returned anymore, so it should be removed from `politeiawww/api/v1/v1.go`.